src/shared/components/results/ResultsData.tsx

```diff
@@ -96,6 +96,33 @@ const ResultsData: FC<{
     return <Loader progress={progress} />;
   }
 
+  const dataTableWithLoader =
+    namespace !== Namespace.idmapping ? (
+      <DataTableWithLoader
+        getIdKey={getIdKey}
+        columns={columns}
+        data={allResults}
+        loading={loading}
+        selected={selectedEntries}
```

Comment:
In the same way that we call `ResultsDataHeader` with different props to deal with idmapping specific behaviour how about having a `disableRowSelection` prop for `ResultsData`?

Reply:
Not sure, quite like the way it is at the moment in that if you provide one you have to provide the other. I think an extra attribute would be unnecessary, unless we end up having to do this lots of times.

Reply:
I think I've put this comment in the wrong place. This should be about the check at L100:

```
namespace !== Namespace.idmapping ? (
```

and wether or not we should instead do:

```
disableRowSelection ?
```

More broadly, shall we just do:

* the `namespace === 'id-mapping'` check for the all of the exceptional behaviour throughout each of the components

or

* Provide additional disable props throughout (eg `disableCardToggle`) which is activated within `IDMappingResult`

but try to avoid both